TSC-Charter.md

```diff
@@ -50,43 +67,54 @@ to continue attending TSC meetings as an observer.
 
 ## Section 5. Responsibilities of the TSC.  
 
-The TSC is responsible for all technical development within the Planetary Software Organization,
-including:
-
-* Setting release dates.
-* Release quality standards.
-* Technical direction.
-* Project governance and process (including this policy).
-* GitHub repository hosting.
-* Maintaining a list of affiliate projects
-* Conduct guidelines.
-* Maintaining the list of additional Collaborators.
-* Development process and coding standards.
+The TSC exercises autonomy in setting up and maintaining procedures and policies,
+as well as management and administrative structures as it deems appropriate for the
+maintenance and operation of these projects and resources.
+
+The TSC is responsible for all technical development within the
+Planetary Software Organization, including:
+
+* Technical direction
+* Setting and maintaining governance rules for the conduct and make-up of the
+  TSC, Working Groups and other bodies within the TSC's domain
+* Creating new repositories and projects under the 
+  [_planetarysoftware_ GitHub organization](https://github.com/planetarysoftware)
+  as required
+* Maintaining a list of projects and working groups
+* Setting and maintaining appropriate standards for community discourse via the
+  various mediums under TSC control
+* Maintaining the list of additional Collaborators
```

Comment:
Maintaining A list, like above for projects?

Reply:
Oh, this is about grammer, 'a list' or 'the list.'  I can't imagine there would be more than one of either of those, so we could use 'a list' or 'the list.'  I have no preference.  Suggest one or the other, and I'll normalize.

Reply:
I was just thinking if you write "the list" maybe you were thinking it's something known to the reader and I missed that it was introduced elsewhere? Otherwise "a list" I guess is better.

Reply:
Good point, I'll change this to 'a list.'